technologic/board/ts7100-usbprod/scripts/blast.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright (c) 2021-2022 Technologic Systems, Inc. dba embeddedTS
+
+# Edit these variables as needed. When porting, this should be all that
+# needs to change for a new platform. Should be.
+
+# Whole device device node path for eMMC. Assuming it is static each boot.
```

Comment:
I see "device" here twice is that intentional?

Reply:
That is a copy/paste error. I bet every other `blast.sh` has that. I'll open an issue about it, thanks.

Reply:
makes sense to me.